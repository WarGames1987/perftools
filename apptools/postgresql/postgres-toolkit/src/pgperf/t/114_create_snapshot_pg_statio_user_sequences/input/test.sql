CREATE SEQUENCE s1;
SELECT nextval('s1');

SELECT pgperf.create_snapshot_pg_statio_user_sequences(0);
SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';

SELECT pgperf.create_snapshot_pg_statio_user_sequences(1);
SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';

SELECT pgperf.create_snapshot_pg_statio_user_sequences(2);
SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';

SELECT pgperf.delete_snapshot_pg_statio_user_sequences(1);
SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';

SELECT pgperf.delete_snapshot_pg_statio_user_sequences(2);
SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';

SELECT pgperf.delete_snapshot_pg_statio_user_sequences(0);
SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';

DROP SEQUENCE s1;
