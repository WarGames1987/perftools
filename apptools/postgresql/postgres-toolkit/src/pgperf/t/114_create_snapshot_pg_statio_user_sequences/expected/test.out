CREATE SEQUENCE s1;
CREATE SEQUENCE
SELECT nextval('s1');
 nextval 
---------
       1
(1 row)

SELECT pgperf.create_snapshot_pg_statio_user_sequences(0);
 create_snapshot_pg_statio_user_sequences 
------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';
 count 
-------
     1
(1 row)

SELECT pgperf.create_snapshot_pg_statio_user_sequences(1);
 create_snapshot_pg_statio_user_sequences 
------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';
 count 
-------
     2
(1 row)

SELECT pgperf.create_snapshot_pg_statio_user_sequences(2);
 create_snapshot_pg_statio_user_sequences 
------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';
 count 
-------
     3
(1 row)

SELECT pgperf.delete_snapshot_pg_statio_user_sequences(1);
 delete_snapshot_pg_statio_user_sequences 
------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';
 count 
-------
     2
(1 row)

SELECT pgperf.delete_snapshot_pg_statio_user_sequences(2);
 delete_snapshot_pg_statio_user_sequences 
------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';
 count 
-------
     1
(1 row)

SELECT pgperf.delete_snapshot_pg_statio_user_sequences(0);
 delete_snapshot_pg_statio_user_sequences 
------------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_statio_user_sequences WHERE schemaname <> 'pgperf';
 count 
-------
     0
(1 row)

DROP SEQUENCE s1;
DROP SEQUENCE
